File: Бодлогууд/Лаб9/JobDescription.cpp
#include "JobDescription.h"

Job::Job(){
             this->description = new char[strlen("unknown") + 1];
            strcpy(this->description, "unknown");
        }
        Job::Job(char *description){
            this->description = new char[strlen(description) + 1];
            strcpy(this->description, description);
        }
        char* Job::getDescription(){
            return this->description;
        }
        void Job::setDescription(char* description){
            delete this->description;
            this->description = new char[strlen(description) + 1];
            strcpy(this->description, description);
        }
        Job::~Job(){
            delete this->description;
        }

File: Бодлогууд/untitled/src/main/java/org/example/JobDescription.java
package org.example;

import javax.swing.*;

public class JobDescription {
    private  String Description;
    JobDescription(){
        Description = "Unnamed";
    }
    JobDescription(String Description){
         this.Description = Description;
    }
    String getDescription(){
         return Description;
    }
    void setDescription(String Description){
        this.Description = Description;
    }
}
